ci/jenkins/stop-jobs.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+PULL_REQUEST_ID=""
+JENKINS_URL=""
+DEFAULT_WORKDIR="/var/lib/jenkins"
```

Comment:
instead of "/var/lib/jenkins" you can use "HOME" env variable, because in future if the home directory is changed you will need to update the code also. HOME is the env variable available there.